modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/form/validation/util/DateParameterUtil.java

```diff
@@ -47,8 +52,21 @@ public static LocalDate getLocalDate(String dateString) {
 			return null;
 		}
 
-		return LocalDate.parse(
-			dateString, DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+		try {
+			SimpleDateFormat simpleDateFormat = new SimpleDateFormat(
+				_DATE_FORMAT);
+
+			return LocalDate.parse(
+				simpleDateFormat.format(
+					DateUtil.parseDate(
+						_DATE_FORMAT, dateString, LocaleUtil.getDefault())),
+				DateTimeFormatter.ofPattern(_DATE_FORMAT));
```

Comment:
Hi @seltong, let's try to find a better way to convert this date to the pattern date.